src/applications/vre/definitions/profileAddress.js

```diff
@@ -0,0 +1,290 @@
+/**
+ * This uiSchema is modeled after how addresses are handled in the Profile application, and makes the same pattern
+ * available for use inside forms generated by the platform's form system.
+ */
+
+import React from 'react';
+import AdditionalInfo from '@department-of-veterans-affairs/formation-react/AdditionalInfo';
+import get from 'platform/utilities/data/get';
+import {
+  countries,
+  states50AndDC,
+  militaryCities,
+  militaryStates,
+} from 'vets-json-schema/dist/constants.json';
+
+/**
+ * CONSTANTS:
+ * 2. USA - references USA value and label
+ * 3. MilitaryBaseInfo - React component. Wrapped in AdditionalInfo component and used as description
+ */
+
+/**
+ * PATTERNS
+ * STREET_PATTERN - rejects white space only
+ * US_POSTAL_CODE_PATTERN - Matches 5 digit zipcodes and zip+4 patterns: 12345, 12345-1234
+ */
+const STREET_PATTERN = '^.*\\S.*';
```

Comment:
I have seen a lot of different implementations for reg ex patterns used throughout the VA.gov stack. I wonder if we can audit all of these and come up with one set that can be used everywhere?